grzesiek/src/test/java/school/testautomation/page/RegisterErrorPage.java

```diff
@@ -0,0 +1,26 @@
+package school.testautomation.page;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class RegisterErrorPage extends RegisterPage {
```

Comment:
To ciekawe zjawisko.
Taki page, niepage co ma metody, ktore nigdzie nie prowadza...
One way ticket :)
Mysle ze te 2 pola moglbys dodac spokojnie do RegisterPage i stamtad je wolac do asercji.

Ale tak tez zle nie jest - grunt ze zrobione ze smakiem i kazdy rozumie o co chodzi, gdy czyta ten kod.